app/assets/stylesheets/modules/_flow-grid.scss

```diff
@@ -0,0 +1,73 @@
+// These two constants must match those in flow_grid.js.coffee
+$grid-max: 30;
+$cell-max: 8;
+
+$cell-min: 4;
+$grid-min: $grid-max - $cell-max + 1;
+
+@mixin stellar-flow() {
+  @for $g from $grid-min through $grid-max {
+    .of-#{$g} {
+      padding-top: (100% / $g) * 6;
+    }
+    @for $c from $cell-min through $cell-max {
+      .c-#{$c}.of-#{$g} {
+        width: (100% / $g) * $c;
+      }
+    }
+  }
+}
+
+.flow-grid {
+  padding: 5px;
+
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+}
+
+@include stellar-flow();
+
+.grid-cell {
+  position: relative;
+  float: left;
+  box-sizing: border-box;
+  background-color: #ccc;
+  overflow: hidden;
+  border: 5px solid white;
+}
+
+.is-selected {
+  border: 5px solid red;
+}
+
+.grid-item {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.cell-details {
+  display: none;
+
+  .cell-details-container & {
+    display: block;
+  }
+}
+
+.cell-details-container {
+  background-color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.